Node panics with config missing committee

```
% cat config2.yml 
#

ApplicationConfiguration:
  DBConfiguration:
    BoltDBOptions:
      FilePath: /var/lib/neogo/qwe/morph.bolt
    Type: boltdb
ProtocolConfiguration:
  InitialGASSupply: 52000000
  Magic: 735783775
  MaxTraceableBlocks: 2102400
  MemPoolSize: 50000
  P2PSigExtensions: true
  StateRootInHeader: false
  TimePerBlock: 15s
  VerifyTransactions: true
```
```
% neogo-testnet-rpc db dump --config-file=config2.yml --out dump
2023-12-01T10:02:52.887Z	INFO	P2PNotaryRequestPayloadPool size is not set or wrong, setting default value	{"P2PNotaryRequestPayloadPoolSize": 1000}
2023-12-01T10:02:52.887Z	INFO	MaxBlockSize is not set or wrong, setting default value	{"MaxBlockSize": 262144}
2023-12-01T10:02:52.887Z	INFO	MaxBlockSystemFee is not set or wrong, setting default value	{"MaxBlockSystemFee": 900000000000}
2023-12-01T10:02:52.887Z	INFO	MaxTransactionsPerBlock is not set or wrong, using default value	{"MaxTransactionsPerBlock": 512}
2023-12-01T10:02:52.887Z	INFO	MaxValidUntilBlockIncrement is not set or wrong, using default value	{"MaxValidUntilBlockIncrement": 5760}
2023-12-01T10:02:52.887Z	INFO	Hardforks are not set, using default value
2023-12-01T10:02:52.888Z	INFO	restoring blockchain	{"version": "0.2.10"}
panic: runtime error: integer divide by zero

goroutine 1 [running]:
github.com/nspcc-dev/neo-go/pkg/config.(*ProtocolConfiguration).ShouldUpdateCommitteeAt(...)
	github.com/nspcc-dev/neo-go/pkg/config/protocol_config.go:207
github.com/nspcc-dev/neo-go/pkg/core/native.(*NEO).InitializeCache(0xc000344b00, 0x28c970, 0x70c928000000dd40?)
	github.com/nspcc-dev/neo-go/pkg/core/native/native_neo.go:349 +0x54a
github.com/nspcc-dev/neo-go/pkg/core.(*Blockchain).initializeNativeCache(0xc000344dc0, 0x28c970?, 0x1?)
	github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:1035 +0xc4
github.com/nspcc-dev/neo-go/pkg/core.(*Blockchain).init(0xc000344dc0)
	github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:483 +0x72e
github.com/nspcc-dev/neo-go/pkg/core.NewBlockchain({0x1490d98?, 0xc00030e010}, {{0x0, {0x0, 0x0}, 0x2bdb2b5f, 0xc350, 0xc0003b25d0, 0x12795f58d50000, 0x3e8, ...}, ...}, ...)
	github.com/nspcc-dev/neo-go/pkg/core/blockchain.go:327 +0x1838
github.com/nspcc-dev/neo-go/cli/server.initBlockChain({{0x0, {0x0, 0x0}, 0x2bdb2b5f, 0xc350, 0x0, 0x12795f58d50000, 0x0, 0x0, 0x0, ...}, ...}, ...)
	github.com/nspcc-dev/neo-go/cli/server/server.go:656 +0x205
github.com/nspcc-dev/neo-go/cli/server.initBCWithMetrics({{0x0, {0x0, 0x0}, 0x2bdb2b5f, 0xc350, 0x0, 0x12795f58d50000, 0x0, 0x0, 0x0, ...}, ...}, ...)
	github.com/nspcc-dev/neo-go/cli/server/server.go:135 +0x58
github.com/nspcc-dev/neo-go/cli/server.dumpDB(0xc00036a9a0)
	github.com/nspcc-dev/neo-go/cli/server/server.go:183 +0x527
github.com/urfave/cli.HandleAction({0xffa480?, 0x1332e38?}, 0x4?)
	github.com/urfave/cli@v1.22.5/app.go:524 +0x50
github.com/urfave/cli.Command.Run({{0x11dad52, 0x4}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x121475e, 0x30}, {0x122cb60, ...}, ...}, ...)
	github.com/urfave/cli@v1.22.5/command.go:173 +0x63e
github.com/urfave/cli.(*App).RunAsSubcommand(0xc00035f880, 0xc00036a6e0)
	github.com/urfave/cli@v1.22.5/app.go:405 +0xdc7
github.com/urfave/cli.Command.startApp({{0x11da5ce, 0x2}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x11efec0, 0x16}, {0x0, ...}, ...}, ...)
	github.com/urfave/cli@v1.22.5/command.go:372 +0xb58
github.com/urfave/cli.Command.Run({{0x11da5ce, 0x2}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x11efec0, 0x16}, {0x0, ...}, ...}, ...)
	github.com/urfave/cli@v1.22.5/command.go:102 +0x7e5
github.com/urfave/cli.(*App).Run(0xc00035f6c0, {0xc00014e120, 0x6, 0x6})
	github.com/urfave/cli@v1.22.5/app.go:277 +0xb07
main.main()
	./main.go:12 +0x2d
zsh: exit 2     neogo-testnet-rpc db dump --config-file=config2.yml --out dump
```

```
% neogo-testnet-rpc --version
NeoGo
Version: 0.104.0
GoVersion: go1.21.4
```

--- a/pkg/config/protocol_config.go
+++ b/pkg/config/protocol_config.go
@@ -106,8 +106,12 @@ func (p *ProtocolConfiguration) Validate() error {
 			shouldBeDisabled = true
 		}
 	}
-	if p.ValidatorsCount != 0 && len(p.ValidatorsHistory) != 0 {
-		return errors.New("configuration should either have ValidatorsCount or ValidatorsHistory, not both")
+	if p.ValidatorsCount != 0 && len(p.ValidatorsHistory) != 0 || p.ValidatorsCount == 0 && len(p.ValidatorsHistory) == 0 {
+		return errors.New("configuration should either have one of ValidatorsCount or ValidatorsHistory, not both")
+	}
+
+	if len(p.StandbyCommittee) == 0 {
+		return errors.New("configuration should include StandbyCommittee")
 	}
 	if len(p.StandbyCommittee) < int(p.ValidatorsCount) {
 		return errors.New("validators count can't exceed the size of StandbyCommittee")

--- a/pkg/config/protocol_config_test.go
+++ b/pkg/config/protocol_config_test.go
@@ -135,6 +135,20 @@ func TestProtocolConfigurationValidation(t *testing.T) {
 		ValidatorsHistory: map[uint32]uint32{0: 1, 100: 4},
 	}
 	require.NoError(t, p.Validate())
+	p = &ProtocolConfiguration{
+		StandbyCommittee:  []string{},
+		CommitteeHistory:  map[uint32]uint32{0: 1, 100: 4},
+		ValidatorsHistory: map[uint32]uint32{0: 1, 100: 4},
+	}
+	err := p.Validate()
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "configuration should include StandbyCommittee")
+	p = &ProtocolConfiguration{
+		StandbyCommittee: []string{"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2"},
+	}
+	err = p.Validate()
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "configuration should either have one of ValidatorsCount or ValidatorsHistory, not both")
 }
 
 func TestProtocolConfigurationValidation_Hardforks(t *testing.T) {
@@ -149,13 +163,20 @@ func TestProtocolConfigurationValidation_Hardforks(t *testing.T) {
 			"Aspidochelone": 2,
 			"Basilisk":      1, // Lower height in higher hard-fork.
 		},
+		StandbyCommittee: []string{
+			"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2",
+		},
+		ValidatorsCount: 1,
 	}
 	require.Error(t, p.Validate())
 	p = &ProtocolConfiguration{
 		Hardforks: map[string]uint32{
 			"Aspidochelone": 2,
 			"Basilisk":      2, // Same height is OK.
+		}, StandbyCommittee: []string{
+			"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2",
 		},
+		ValidatorsCount: 1,
 	}
 	require.NoError(t, p.Validate())
 	p = &ProtocolConfiguration{
@@ -163,18 +184,30 @@ func TestProtocolConfigurationValidation_Hardforks(t *testing.T) {
 			"Aspidochelone": 2,
 			"Basilisk":      3, // Larger height is OK.
 		},
+		StandbyCommittee: []string{
+			"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2",
+		},
+		ValidatorsCount: 1,
 	}
 	require.NoError(t, p.Validate())
 	p = &ProtocolConfiguration{
 		Hardforks: map[string]uint32{
 			"Aspidochelone": 2,
 		},
+		StandbyCommittee: []string{
+			"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2",
+		},
+		ValidatorsCount: 1,
 	}
 	require.NoError(t, p.Validate())
 	p = &ProtocolConfiguration{
 		Hardforks: map[string]uint32{
 			"Basilisk": 2,
 		},
+		StandbyCommittee: []string{
+			"02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2",
+		},
+		ValidatorsCount: 1,
 	}
 	require.NoError(t, p.Validate())
 }

